### PR TITLE
Use promises and ES2015ify

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,13 +16,8 @@ process.on('exit', function () {
...
-		cb = opts;
-		opts = {};
-	}
-
-	cb = cb || function () {};
+module.exports = function (opts) {
+	opts = opts || {};
 
 	if (opts.title && opts.title.length > 15) {
 		throw new Error('The title can be maximum 15 characters');
```
Comment (SamVerschueren) [inline index.js]: Return rejected promise.

File: package.json [hunk 2]
```diff
@@ -30,9 +30,20 @@
...
-    "ava": "0.0.4",
+    "ava": "*",
     "fkill": "^1.0.0",
-    "process-exists": "^1.0.0"
+    "pify": "^2.2.0",
+    "process-exists": "^1.0.0",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
```
Comment (SamVerschueren) [inline package.json]: [`ignores`](https://github.com/sindresorhus/xo#ignores) instead of `ignore`.

File: package.json [hunk 3]
```diff
@@ -30,9 +30,20 @@
...
+      "test.js",
+      "test2.js"
```
Comment (sindresorhus) [inline package.json]: `"test*.js"`

File: readme.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ noopProcess(function (err, pid) {
 
 ## API
 
-### noopProcess([options], [callback])
+### noopProcess([options])
 
-Creates a [noop](https://en.wikipedia.org/wiki/NOP) process.
+Creates a [noop](https://en.wikipedia.org/wiki/NOP) process. Returns a promise that resolves a process ID for the created process.
```
Comment (sindresorhus) [inline readme.md]: I've found a short way to describe promises:

> Returns a promise for the process ID of the created process.

File: test2.js [hunk 5]
```diff
@@ -1,22 +1,13 @@
...
-var processExists = require('process-exists');
-var fkill = require('fkill');
+import test from 'ava';
+import processExists from 'process-exists';
+import fkill from 'fkill';
+import pify from 'pify';
 
-test('persistent option', function (t) {
-	t.plan(5);
+const processExistsP = pify(processExists);
```
Comment (sindresorhus) [inline test2.js]: Can you use version 2.0.0 here?

**Pull request merged**